CONTRIBUTING.md

```diff
@@ -560,60 +560,84 @@ It's mandatory to use *release/* prefix in your branch name.
 
 Example:
 
-* Checkout release branch
+* Checkout release branch:
 
 ```sh
 cd mapbender
 git checkout "release/3.0.6"
 ``` 
-* Fetch changes  
+* Fetch changes:  
 
 ```sh
 git fetch -a
 git pull
 ``` 
-* Merge changes
+* Merge changes:
 
 ```sh
 git merge "hotfix/bug-short-description"
 ``` 
-* If conflicts arise, resolve [them][Resolve git conflicts]
-* Run or add new tests relevant to the new feature
-* Code review
-* Run tests
-* Save changes
+* If conflicts arise, resolve [them][Resolve git conflicts].
+* Run or add new tests relevant to the new feature.
+* Review the code.
+* Run tests.
+* Save changes:
 
 ```sh
 git commit -m "Merge 'hotfix/bug-short-description'"
 ``` 
-* Push on [github]
+* Push on [github]:
 
 ```sh
 git push
 ``` 
 
+# Building packages 
+
+For distributing and building packages, there is a matching [composer] command:
+
+* `bin/composer build` Build package command with follow optional parameters:
+    * **[tar.gz|zip]** - Optional parameter that defines the package file format. The default configuration is defined in `composer.json` as `config/archive-format`.
+    * **[dist-name]** - Optional parameter that defines the package file name prefix. The default configuration is defined in `composer.json` as `name`, a vendor name will be ignored.
+    * **[dist-version]** - Optional parameter that defines the package version. This is included as suffix in the package name. The default configuration is defined in `composer.json` as `version`.
+
+You can define the [composer] distributing path in `composer.json` as `config/archive-dir`. The default location is a `dist` folder located in root of the project. 
+
+## Build package example
+
+Running this command distributes and builds the package `dist/test-distribution.1.0.1.tar.gz`:
```

Comment:
You can build and distribute an artifact to `dist/test-distritution.1.0.1.tar.gz` by running: